source/math/FloatTransform2D.ooc

```diff
@@ -68,6 +68,10 @@ FloatTransform2D: cover {
 	rotationZ ::= this b atan2(this a)
 	inverse: This { get {
 		determinant := this determinant
+		version (safe) {
+			if (determinant == 0)
+				raise("Determinant is zero in FloatTransform2D inverse()")
```

Comment:
The question is `can the matrix be inverted` and you can answer this without computing the determinant, but it's not computationally more efficient in this case. Your explanation makes sense, so I will fix.